db/README.md

```diff
@@ -14,24 +14,37 @@ You can use the scripts in this directory (`db`, `run-proxy`) to connect and int
 
 ### Required Environment Variables
 
-Before starting, ensure you have set the following environment variables:
+Before starting, ensure you have set the following environment variables. The credentials and the cloud SQL database info are available from 1Password's team vault.
 
-- `GCLOUD_AUTH`
 - `CLOUD_SQL_CONNECTION_NAME`
 - `CLOUD_SQL_DATABASE_PORT`
 - `CLOUD_SQL_DATABASE_USERNAME`
 - `CLOUD_SQL_DATABASE_PASSWORD`
 - `CLOUD_SQL_DATABASE_NAME`
+   * The name of the database to use currently. Example: `preflight`.
+- `CLOUD_SQL_CREDENTIALS`
+   * Download cloud-sql-proxy.json from 1Password team vault and save.
+   * The value of this variable is the path to the file.
 
 ### Setup Instructions
 
+1. Download the dependencies: Google Cloud CLI and MySQL v8.
+   ```
+   brew install gcloud
+   brew install mysql@8.4
+   ```
+1. Log in to Google Cloud and configure the project name.
+   ```
+   gcloud auth login
+   gcloud config set project [GCP project name]
+   ```
 1. Download the [Cloud SQL Auth Proxy](https://cloud.google.com/sql/docs/mysql/connect-auth-proxy#install) appropriate for your OS.
-2. Place the binary in this `DB/` directory.
-3. Open a terminal tab and run:
+1. Place the binary in this `DB/` directory.
+1. Open a terminal tab and run:
```

Comment:
Nit:I see your comment above, but now we see each step with 1. , either we have 1., 2., .. or bullet points